### DISCUSSION TITLE
@vueform/vueform theme prop not work

### DISCUSSION THREAD

**Discussion by JunJieWang-Dev** [vueform/vueform#304] [Questions] [answered]
### Environment

@vueform/vueform:  1.9.9
"vue": "3.4.27"
tailwindcss: 3.4.3

### Reproduction

/

### Describe the bug

my code:
```html
<Vueform v-bind="vueform" ref="form$" />
```

```javascript
const formData = {
    ...form,
    ...builder,
    schema,
    theme,
    displayErrors: true,
    addClass: 'vf-custom-form',
  };
  vueform.value = formData;
```

result:
web can render form with schema； but theme prop donot take effect。 the theme value is like: 
```
{ 
bg-addon :  "#e9ecef",
bg-btn:  "var(--vf-primary)",
bg-btn-danger:  "var(--vf-danger)",
bg-btn-secondary:  "var(--vf-gray-200)",
bg-checkbox:  "#ffffff"
......
```

What's wrong? Did I make a mistake ?

**Comment by adamberecz** [comment, marked as answer]
Please refer to this guide on using themes exported by the builder:
https://builder.vueform.com/docs/rendering#theme

**Reply by JunJieWang-Dev** [reply]
Great， That works well
